test/quantization/test_quantize_script.py

```diff
@@ -1672,17 +1669,15 @@ def forward(self, x):
         m1 = convert_script(m, debug=True)
         conv_op_quant = 4
         # NB: This Needs to be updated when we add more ops to test
-        general_value_op_quant = 2
-        FileCheck().check_count("aten::quantize_per_tensor", conv_op_quant + general_value_op_quant + 1, exactly=True) \
+        general_value_op_quant = 12
+        FileCheck().check_count("aten::quantize_per_tensor(", conv_op_quant + general_value_op_quant + 1, exactly=True) \
                    .run(m1.graph)
 
         # This checks that the dequantize from the output of first conv
         # is being propagated to the end, so that we don't insert extra
         # observers and also successfully fused two quantized::conv2d
         # patterns
         # one quantize_per_tensor for input
-        m = wrap_cpp_module(torch._C._jit_pass_insert_observers(
-            torch.jit.script(M())._c, 'forward', {'': qconfig}, inplace=False))
         m2 = convert_script(m, debug=False)
         FileCheck().check_count("aten::quantize_per_tensor(", 1, exactly=True) \
```

Comment:
Can we also check for presence of the pooling ops in the final graph?

Reply:
that is probably not worth it since that would mean spelling out the graph. for example, in the later PR, we have some more complicated ops like interpolate.
What we are checking here is the number of inserted quantize_per_tensor ops are correct, and I think it is enough to ensure the correctness